library/std/src/sys/process/unix/vxworks.rs

```diff
@@ -147,13 +147,17 @@ impl Process {
     }
 
     pub fn kill(&mut self) -> io::Result<()> {
+        self.send_signal(libc::SIGKILL)
+    }
+
+    pub fn send_signal(&mut self, signal: i32) -> io::Result<()> {
```

Comment:
All the `send_signal` functions (and `kill` if they exist) need to change the `&mut` to `&`